src/main/java/school/lemon/changerequest/java/FractionNumberDemo.java

```diff
@@ -0,0 +1,25 @@
+package school.lemon.changerequest.java;
+
+/**
+ * Created by lera on 14.12.16.
+ */
+public class FractionNumberDemo {
+    public static void main(String[] args) {
+        FractionNumber fractionNumber1 = new FractionNumber(3, 9);
+        FractionNumber fractionNumber2 = new FractionNumber(1, 6);
+        //    FractionNumber fractionNumber3 = new FractionNumber(1,0);
+        System.out.println("FractionNumber1 = " + fractionNumber1.toString());
```

Comment:
There is no need to call `toString` method each time, cause regarding to concatenation with String rules - `toString` method would be called automatically.